plugins/inputs/opcua_listener/sample.conf

```diff
@@ -58,13 +58,43 @@
   ## identifier_type   - OPC UA ID type (s=string, i=numeric, g=guid, b=opaque)
   ## identifier        - OPC UA ID (tag as shown in opcua browser)
   ## default_tags      - extra tags to be added to the output metric (optional)
+  ## monitoring_params - additional settings for the monitored node (optional)
+  ##
+  ## Monitoring parameters
+  ## sampling_interval  - interval at which the server should check for data
+  ##                      changes (default: 0)
```

Comment:
What does a default of 0 mean? Whatever the server decides?

Reply:
Exactly. The current implementation sends 0 as well so I didn't want to break that.